### PR TITLE
[JENKINS-67496] Drag & drop is messed up, drag placeholders always have tiny size

### PR DESCRIPTION
**Pull request by benebsiny**
See [JENKINS-67496](https://issues.jenkins-ci.org/browse/JENKINS-67496).

Keep the height when we drag the div. See demo:

Animation

### Proposed changelog entries

* Keep the same height when dragging and dropping a component (regression in 2.277).

### Proposed upgrade guidelines

N/A

### Submitter checklist

- [X] (If applicable) Jira issue is well described
- [X] Changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developer, depending on the change). [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
  * Fill-in the `Proposed changelog entries` section only if there are breaking changes or other changes which may require extra steps from users during the upgrade
- [X] Appropriate autotests or explanation to why this change has no tests
- [ ] For dependency updates: links to external changelogs and, if possible, full diffs

### Desired reviewers

@mention

### Maintainer checklist

Before the changes are marked as `ready-for-merge`: 

- [ ] There are at least 2 approvals for the pull request and no outstanding requests for change
- [ ] Conversations in the pull request are over OR it is explicit that a reviewer does not block the change
- [ ] Changelog entries in the PR title and/or `Proposed changelog entries` are correct
- [ ] Proper changelog labels are set so that the changelog can be generated automatically
- [ ] If the change needs additional upgrade steps from users, `upgrade-guide-needed` label is set and there is a `Proposed upgrade guidelines` section in the PR title. ([example](https://github.com/jenkinsci/jenkins/pull/4387))
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins-ci.org/issues/?filter=12146)).

### CODE REVIEW

File: war/src/main/js/sortable-drag-drop.js
Comment (daniel-beck) [inline war/src/main/js/sortable-drag-drop.js:29]: We claim to still support IE 11 per https://www.jenkins.io/doc/administration/requirements/web-browsers/. If this is https://caniuse.com/template-literals, then this won't work.

**Review by daniel-beck — APPROVED** [review]
Works well, thanks for taking care of this!

Approval conditional on IE11 compatibility (:sob:)

**Review by janfaracik — APPROVED** [review]

File: war/src/main/js/sortable-drag-drop.js
Comment (janfaracik) [reply war/src/main/js/sortable-drag-drop.js:29]: Off topic but is there a date in mind for when IE11 support will be dropped?

Comment (daniel-beck) [reply war/src/main/js/sortable-drag-drop.js:29]: No idea. Something for the UX sig I guess?

Comment (timja) [reply war/src/main/js/sortable-drag-drop.js:29]: I've added to the agenda of the next meeting on Wednesday https://docs.google.com/document/d/1QttPwdimNP_120JukigKsRuBvMr34KZhVfsbgq1HFLM/edit#heading=h.5w6rztvl5s02

Comment (timja) [reply war/src/main/js/sortable-drag-drop.js:29]: and mailing list: https://groups.google.com/g/jenkinsci-dev/c/piANoeohdik/m/PewPuvLSAAAJ

**Review by NotMyFault — APPROVED** [review]
Works great based on my testing 👏🏻

**Comment by timja** [comment]
We've dropped IE 11 support now, this won't be eligible for backporting though, (unless someone makes it compatible)

----

This PR is now ready for merge, after ~24 hours, we will merge it if there's no negative feedback.

Thanks!

**Pull request merged**
